ta4j-examples/pom.xml

```diff
@@ -1,48 +1,210 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <parent>
-        <groupId>org.ta4j</groupId>
-        <artifactId>ta4j-parent</artifactId>
-        <version>0.15-SNAPSHOT</version>
-    </parent>
-    <artifactId>ta4j-examples</artifactId>
-
-    <name>Ta4j Examples</name>
-    <description>Provides examples that demonstrate how to use ta4j</description>
-
-    <dependencies>
-        
-        <!-- Test dependencies -->
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.13.1</version>
```

Comment:
copy

Reply:
btw, normally you do not need to edit `pom.xml`.

Reply:
I did not edit the examples/pom; at least not deliberately.  It and the changelog were modified for the 0.14 release.  

BTW, the new examples POM doesn't work as it is - at least not for me.  It cannot find core 0.15-SNAPSHOT; I think there was an issue with the new junit version too.  Perhaps your build is inserting the SNAPSHOT version for development and the released one is fine.  But as it is, I will need to edit the POM to make it work and be careful not to commit it.  I can adapt.